ci/benchmarks/partial-conv/evo2_pretrain.yaml

```diff
@@ -55,7 +62,6 @@ script: |-
   --enable-preemption \
   --ckpt-async-save \
   --use-megatron-comm-overlap-llama3-8k \
-  --use-subquadratic_ops \
```

Comment:
Ensure the evo2 train command uses `--use-subquadratic_ops` or add it as an ops keyword: '--use-subquadratic_ops' in the product section. 
```
....
  products:
    - model_type: hyena
      config_name: 1b
      ops_kwargs: "--use-subquadratic_ops"
    - model_type: mamba
      config_name: hybrid_mamba_8b
      ops_kwargs: ""
script: |-
.....
${ops_kwargs}
```

Reply:
Maybe evo2 mamba can use it as well, but lets first have one training working without